pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mkdocs-multirepo-plugin"
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
Can you revert this change? I'm going to bump the version and generate a new release after this is merged. I'd like to include other changes. Once you revert this I'll merge. Thanks for contributing by the way!

Reply:
@jdoiro3 done, thanks :)